backend/webapi/Infrastructure/HttpClients/BCProvider/BCProviderApiDefinitions.cs

```diff
@@ -52,7 +52,7 @@ public static BCProviderAttributes FromNewUser(string clientId, NewUserRepresent
     /// <summary>
     /// A comma-separated list containing the CPN(s) of Parties Endorsing the User, who have licences from the College of Physicians and Surgeons and the College of Nurses and Midwives.
     /// </summary>
-    public BCProviderAttributes SetEndorserData(IEnumerable<string> endorserData) => this.SetProperty(nameof(endorserData), string.Join(",", endorserData));
+    public BCProviderAttributes SetEndorserData(IEnumerable<string> endorserData) => this.SetProperty(nameof(endorserData), "[" + string.Join(",", endorserData.Select(s => $"\"{s}\"")) + "]");
```

Comment:
ON first review it looks like the escaped double quotes are wrong. But they are not.